src/responder/manager.rs

```diff
@@ -64,6 +64,18 @@ pub fn run() {
                     .guard(guard::Delete())
                     .to(routes::reporter_flush),
             )
+            .service(
+                web::resource("/probe/start_planned_maintenance/{probe_id}")
+                    .wrap(middleware_auth.clone())
+                    .guard(guard::Post())
+                    .to(routes::start_planned_maintenance),
+            )
+            .service(
+                web::resource("/probe/stop_planned_maintenance/{probe_id}")
+                    .wrap(middleware_auth.clone())
+                    .guard(guard::Post())
```

Comment:
should that not be a `Delete` like with the reporter flush endpoint? also the resource endpoint should probably be `/maintenance/{probe_id}`, similar to the reporter endpoint